lib/grainstore/mml_store.js

```diff
@@ -19,11 +17,11 @@ function MMLStore(options) {
 
   options = options || {};
   options.cachedir = options.cachedir || '/tmp/millstone';
-  options.renderer = options.use_workers ? new WorkerRenderer() : new InlineRenderer();
```

Comment:
I would love to keep this as an optional feature, so we can decide to use workers.